frontend/components/sections/QuoteSection.tsx

```diff
@@ -41,8 +41,10 @@ export function QuoteSection({ id, quote, author, image }: QuoteSectionProps) {
                <ResponsiveImage
                   src={image.url}
                   alt=""
-                  layout="fill"
-                  objectFit="cover"
+                  fill
```

Comment:
I see that in some places we include `sizes="100vw"` with `fill`, and in others we don't.

The [docs](https://nextjs.org/docs/pages/api-reference/components/image#sizes) say

> If you don't specify a sizes value in an image with the fill property, a default value of 100vw (full screen width) is used.

Is the motivation for including `sizes="100vw"` to be explicit about the value of this property?

Reply:
The image migration script offered by Next was not running smoothly on our project. More in detail some folders were skipped or errored out, so many parts were migrated manually. These explicit properties are probably due to the fact that they were made explicit by the script. On the contrary other sections that were handled manually do not show this property when it is unnecessary.
I'm going to remove that.